### DISCUSSION TITLE
Liquidsoap update

### DISCUSSION THREAD

**Discussion by d-trax** [AzuraCast/AzuraCast#4713] [Q&A]
Hi, is there a way to update Liquidsoap manually without using the updater of AzuraCast?

I'm using AzuraCast with Ansible.

**Comment by ItsMitchh** [comment]
We're currently working on adding LS 2.0. Adding it retroactively through an unsupported update may cause even more problems to your installation and it's something we won't recommend. You can keep an eye on our progress in our Pull Request section.

**Comment by d-trax** [comment]
Oh, that's amazing news! Wish you all the best guys! :)
